Sources/App/Commands/Analyzer.swift

```diff
@@ -57,18 +81,27 @@ func analyze(application: Application, limit: Int) throws -> EventLoopFuture<Voi
 
     // FIXME: sas 2020-05-04: Workaround for partial flush described here:
     // https://discordapp.com/channels/431917998102675485/444249946808647699/706796431540748372
+    // FIXME: this breaks exposing this via the API:
+    // Precondition failed: BUG DETECTED: wait() must not be called when on an EventLoop.
```

Comment:
I’ll add an issue to track this, maybe someone can jump in to give a hand.

Reply:
I'm not sure I can be much help here, but let's raise an issue for it so it doesn't get forgotten after this is merged.

Reply:
Ah yes, thanks for the reminder to actually paste that in :D

#29